src/client/client.ts

```diff
@@ -628,11 +672,10 @@ export class Client {
               ]);
               logger.debug(`[WD] c:"${this.getKey()}" unwatches`);
 
-              if (
-                err instanceof ConnectError &&
-                err.code != ConnectErrorCode.Canceled
-              ) {
+              if (this.isRetryableConnectError(err)) {
```

Comment:
@hackerwins 
If the watchLoop stops due to this condition, when should it restart?